modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/twostep/GridMapQueryExecutor.java

```diff
@@ -771,7 +771,10 @@ else if (log.isInfoEnabled())
         catch (Exception e) {
             e.addSuppressed(err);
 
-            U.error(log, "Failed to send error message.", e);
+            if (node.isClient())
+                log.debug("Failed to send error message: " + X.getFullStackTrace(e));
```

Comment:
why not warning?

Reply:
We were trying to send an error message to a client. This did not work, probably because the client has left, so it's not something that needs to be highlighted in the **server** logs. Hence, it seems that WARN is too much here.

WDYT?